doc/getting_started/index.rst

```diff
@@ -3,319 +3,372 @@
-   pip3 install --user -U west
+         The ``--user`` option puts installed packages into your
+         ``~/.local/bin folder``.  It's very important you permanently update the
+         PATH environment variable with the location of these packages (in your
+         .bashrc profile, as shown).
 
-   # macOS (Terminal) and Windows (cmd.exe)
-   pip3 install -U west
+         |p|
 
-See :ref:`west-install` for additional details on installing west.
+      #. Verify the version of cmake installed on your system using ``cmake
+         --version``, and if it's not version 3.13.1 or higher, download
+         and update cmake:
 
-.. _clone-zephyr:
+         .. code-block:: bash
+
+            cd ~
+            wget https://github.com/Kitware/CMake/releases/download/v3.15.3/cmake-3.15.3-Linux-x86_64.sh
+            chmod +x cmake-3.15.3-Linux-x86_64.sh
+            sudo ./cmake-3.15.3-Linux-x86_64.sh --skip-license --prefix=/usr/local
+            hash -r
+
+   .. group-tab:: macOS
+
+      #. On macOS, install :program:`Homebrew` by following instructions on the `Homebrew
+         site`_, and as shown here. Homebrew is a free and open-source package management system that
+         simplifies installing software on macOS.  While installing Homebrew,
+         you may be prompted to install additional missing dependencies; please follow
+         any such instructions as well.
+
+         .. code-block:: bash
+
+            /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+
+      #. Then, install these host dependencies with the ``brew`` command:
+
+         .. code-block:: bash
+
+            brew install cmake ninja gperf ccache dfu-util qemu dtc python3
+
+      #. Install needed python packages with these commands:
+
+         .. code-block:: bash
+
+            pip3 install -r https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/master/scripts/requirements.txt
+
+      .. _Homebrew site: https://brew.sh/
+
+   .. group-tab:: Windows
+
+      .. note:: If you're running Windows 10 (version 1809 or later)
+         you can run Ubuntu binaries natively within the built-in Windows
+         Subsystem for Linux (WSL). Using WSL will also let you to use
+         the Zephyr SDK. `Install the Windows Subsystem for Linux (WSL)
+         <https://msdn.microsoft.com/en-us/commandline/wsl/install_guide>`__, and
+         then follow the Linux (Ubuntu) Getting Started instructions.
+
+      These instructions assume you are using the Windows ``cmd.exe``
+      command prompt. Some of the details, such as setting environment
+      variables, may differ if you are using PowerShell.
+
+      An easy way to install native Windows dependencies is to first install
+      `Chocolatey`_, a package manager for Windows.  If you prefer to install
+      dependencies manually, you can also download the required programs from their
+      respective websites and verify they can be found on your PATH.
```

Comment:
Does chocolatey add the tools to the PATH? If I remember correctly, it just gives you an output of what to add to the PATH. In that case, we should add a step as a reminder to actually do that.